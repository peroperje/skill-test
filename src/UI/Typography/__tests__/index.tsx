import React from 'react';
import { create } from 'react-test-renderer';

import { Typography } from '../index';

describe('Typography', () => {
  it('Snapshot', () => {
    const component = create(<Typography>Some text</Typography>);
    expect(component.toJSON()).toMatchSnapshot();
  });
  it('Snapshot variant', () => {
    const component = create(<Typography variant="h3">Some text</Typography>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
