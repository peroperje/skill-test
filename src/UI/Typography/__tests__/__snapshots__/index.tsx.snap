// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography Snapshot 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1"
>
  Some text
</p>
`;

exports[`Typography Snapshot variant 1`] = `
<h3
  className="MuiTypography-root MuiTypography-h3"
>
  Some text
</h3>
`;
